test-common/canton/it-lib/src/main/com/daml/CantonFixture.scala

```diff
@@ -69,14 +69,15 @@ trait CantonFixtureWithResource[A]
   //   - temporary file are not deleted (this requires "--test_tmpdir=/tmp/" or similar for bazel builds)
   //   - some debug info are logged.
   //   - output from the canton process is sent to stdout
-  protected val cantonFixtureDebugMode = false
```

Comment:
maybe don't merge with this change

Reply:
Definitely don't merge with this change :)

Reply:
fixed